cvat/apps/engine/user_deletion.py

```diff
@@ -16,21 +15,18 @@
 _USER_DELETION_VALIDATORS = []
 
 
-class AutoRegisterValidatorMeta(ABCMeta):
-    def __new__(mcs, name, bases, namespace):
-        cls = super().__new__(mcs, name, bases, namespace)
-        if not getattr(cls, "__abstractmethods__", None):
-            _USER_DELETION_VALIDATORS.append(cls)
-        return cls
-
-
-class UserDeletionValidator(ABC, metaclass=AutoRegisterValidatorMeta):
+class UserDeletionValidator(ABC):
     """
     Base class for user deletion validators.
     To add a new validator, inherit from this class and implement the `validate` method.
-    The implemented class will be automatically registered and used during user deletion.
+    Then call classmethod 'register_validator()' to enable the class as validator.
     """
 
+    @classmethod
+    def register_validator(cls):
+        if cls not in _USER_DELETION_VALIDATORS:
```

Comment:
Consider changing the type of `_USER_DELETION_VALIDATORS` to `set` if you're trying to repeatedly look for the existing values

Reply:
I think `list` looks more natural here. 
As this will be always short (e.g. not 1000s of values), I hardly think changing to `set` makes sense from algorithm pov